cloudlift/config/environment_configuration.py

```diff
@@ -119,6 +119,7 @@ def _create_config(self):
         cluster_min_instances = prompt("Min instances in cluster", default=1)
         cluster_max_instances = prompt("Max instances in cluster", default=5)
         cluster_instance_type = prompt("Instance type", default='m5.xlarge')
+        cluster_ami_id = prompt("Custom AMI ID for cluster", default='None')
```

Comment:
We should prompt for the map that contains the AMIs to use by region. If the AMI is not present for a region, we fall back to the ecs images provided by AWS.